tests/tasks/test_data_extractor_nlp.py

```
@@ -0,0 +1,368 @@
+"""
+NLP 데이터 추출기 테스트
+
+형태소 분석기를 사용한 키워드 추출이 적용된 NLP 데이터 추출기의 기능을 테스트합니다.
+"""
+import pytest
+import asyncio
```

Ruff (0.11.9)

`asyncio` imported but unused

Remove unused import: `asyncio`

(F401)